itl/command-plugins.conf

```diff
@@ -1382,138 +1382,104 @@ object CheckCommand "disk" {
 			required = true
 			order = -3
 		}
-
 		"-c" = {
 			value = "$disk_cfree$"
 			description = "Exit with CRITICAL status if less than INTEGER units of disk are free or Exit with CRITCAL status if less than PERCENT of disk space is free"
 			required = true
 			order = -3
 		}
-
 		"-W" = {
 			value = "$disk_inode_wfree$"
 			description = "Exit with WARNING status if less than PERCENT of inode space is free"
 		}
-
 		"-K" = {
 			value = "$disk_inode_cfree$"
 			description = "Exit with CRITICAL status if less than PERCENT of inode space is free"
 		}
-
 		"-p" = {
 			value = "$disk_partitions$"
 			description = "Path or partition (may be repeated)"
 			repeat_key = true
 			order = 1
 		}
-
-		"-p_old" = {
-			key = "-p"
-			value = "$disk_partition$"
-			order = 1
-		}
-
```

Comment:
Please leave them inside the configuration, there's a deprecation note in the docs, but it does not hurt for the time being.